lib/instrumentation/http-shared.js

```diff
@@ -24,8 +24,10 @@ exports.instrumentRequest = function (agent, moduleName) {
           agent._instrumentation.currentTransaction = null
         } else {
           var traceparent = req.headers['elastic-apm-traceparent'] || req.headers.traceparent
+          var tracestate = req.headers['elastic-apm-tracestate'] || req.headers.tracestate
           var trans = agent.startTransaction(null, null, {
-            childOf: traceparent
+            childOf: traceparent,
+            tracestate: tracestate
           })
```

Comment:
Here's where we parse either the `elastic-apm-tracestate` or `tracestate` header (preferring the former) looking for `tracestate` values, and then pass them along to the `startTranasction` method.